configure_settings.py

```diff
@@ -1,17 +0,0 @@
-#!/usr/bin/env python
```

Comment:
This file doesn't exist anymore. Don't believe there's really any need for dynamically generating settings, it's always the same anyways.